libsigner/src/runloop.rs

```diff
@@ -75,7 +78,17 @@ pub trait SignerRunLoop<R> {
                     return None;
                 }
             };
-            if let Some(final_state) = self.run_one_pass(next_event_opt) {
+            //info!("SignerRunLoop::main_loop: command_recv");
+            let next_command_opt = match command_recv.recv_timeout(poll_timeout) {
+                Ok(cmd) => Some(cmd),
+                Err(RecvTimeoutError::Timeout) => None,
+                Err(RecvTimeoutError::Disconnected) => {
+                    info!("Command receiver disconnected");
+                    return None;
+                }
+            };
+            //info!("SignerRunLoop::main_loop: run_one_pass");
```

Comment:
Before merging, can you remove the commented-out debug statements?

Reply:
Done.